src/lib/Map.svelte

```diff
@@ -126,7 +126,7 @@
       });
       addPinLayer(map, activeMarkerLayerId, activeMarkerSourceId, 'add-marker');
 
-      map.on('click', markerLayerId, function (e) {
+      map.on('click', markerLayerId, function (e: MapMouseEvent) {
```

Comment:
This is a funny one because it doesn't make sense for the type to be `MapMouseEvent` because we access its `features` key two lines later. MapMouseEvent doesn't have a features key so adding this type causes type issues. I'm not sure what type it should be though.